src/coffee.js

```diff
@@ -0,0 +1,312 @@
+const coffee = [
```

Comment:
Why is this in two files (`coffee.js` and `coffeeObject.js`)? Can we have it in only one file?

Reply:
I thought they are two different data, coffee.js is all data of the coffees and coffeeObject.js is the data we will use for only making the questions, but I can see what you mean, I am little bit afraid to have it in one file, just I am thinking to not make a mess

Reply:
I'm not sure what you mean. We need data about the coffee. We only need it in one place.